rbx_binary/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # rbx_binary Changelog
 
 ## Unreleased
-* Fixed null Refs not being written for unserialized instances. [#184][pr-184]
+* Fixed null Refs not being written for unserialized instances.
```

Comment:
Having a link here would be good. You need to add a `[pr-184]: https://github.com/rojo-rbx/rbx-dom/pull/184` note at the bottom of the section just like we do for the other PRs.